CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**4.0.1 - 03/21/25**
+
+ - Feature: Update required resources for all components
```

Comment:
This isn't a backwards incompatible change?

Reply:
So 4.1.0?

Reply:
My bad, I misread the diff